bayesian/api_v1.py

```diff
@@ -347,35 +347,42 @@ def get(external_request_id):
                 started_at = stack_result["task_result"]["_audit"]["started_at"]
                 finished_at = stack_result["task_result"]["_audit"]["ended_at"]
 
-
         if reco_result is not None and 'task_result' in reco_result:
-            if reco_result["task_result"] != None:
-                recommendation = reco_result['task_result']['recommendations']
+                if reco_result["task_result"] != None:
+                    recommendation = reco_result['task_result']['recommendations']
 
         # Populate sentiment score for packages in user's stack
         if stack_result is not None:
-            user_stack_deps = stack_result['task_result']['user_stack_info']['dependencies']
-            for dep in user_stack_deps:
-                if user_stack_sentiment_result is not None:
-                    dep['sentiment']['overall_score'] = user_stack_sentiment_result['task_result'][dep['name']]['score']
-                else:
-                    dep['sentiment'] = {}
+            user_stack_deps = stack_result.get('task_result').get('user_stack_info').get('dependencies',[])
+            if user_stack_deps:
+                for dep in user_stack_deps:
+                    if user_stack_sentiment_result is not None:
+                        if user_stack_sentiment_result.get('task_result').get(dep['name']) is not None:
```

Comment:
The value of `user_stack_sentiment_result.get('task_result')` can be kept in a new variable e.g. `user_stack_sentiment` so that we can avoid fetching the `task_result` value again and again. This is also more readable.